pkg/processor/service/service.go

```diff
@@ -25,9 +25,9 @@ spec:
   type: {{ .Values.%[1]s.type }}
   selector:
 %[2]s
-  {{- include "%[3]s.selectorLabels" . | nindent 4 }}
+    {{- include "%[3]s.selectorLabels" . | nindent 4 }}
   ports:
-	{{- .Values.%[1]s.ports | toYaml | nindent 2 }}`
+  {{- .Values.%[1]s.ports | toYaml | nindent 2 }}`
```

Comment:
`-` in `{{-` will trim all whitespaces in the left (see [go templates doc](https://pkg.go.dev/text/template#hdr-Text_and_spaces)) and actual intendation is set explicitly in `| nindent 2` part. So  here whitespaces on the left from template exist only for readability and not affect produced yaml.

Reply:
@arttor Indeed, the final YAML will be fine. However, the template of the Helm charts will still have the `tab` in there.

Reply:
ok, but what is the problem with helm template? do you have issues with some linter?

Reply:
The template is fine because the `{{- ` will remove the tab, but it feels weird to have a tab there in the first place.

> Templates should be indented using two spaces (never tabs).
>
> https://helm.sh/docs/chart_best_practices/templates/#formatting-templates

Reply:
i see. can you please also update chart examples by runnig:
```shell
cat test_data/sample-app.yaml | go run ./cmd/helmify examples/app
```
and
```shell
cat test_data/k8s-operator-kustomize.output | go run ./cmd/helmify examples/operator
```
and push changes to your branch?